Sources/SkipBluetooth/Utility.swift

```diff
@@ -0,0 +1,213 @@
+import Foundation
+
+#if SKIP
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.SideEffect
+import android.content.pm.PackageManager
+import android.Manifest
+import android.bluetooth.BluetoothGatt
+import android.bluetooth.BluetoothGattCallback
+import android.bluetooth.BluetoothGattCharacteristic
+import android.bluetooth.BluetoothGattDescriptor
+import android.bluetooth.BluetoothProfile
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+
+// MARK: Globals
+let CCCD = "00002902-0000-1000-8000-00805f9b34fb"
+
+
+/// Prompts the user to allow bluetooth permissions
+///
+/// This process is handled automatically by CoreBluetooth for Swift when you instantiate either
+/// `CBCentralManager` or `CBPeripheralManager`, but Android requires the request
+/// to occur within an activity.
+@Composable
```

Comment:
Is this the only reason the package relies on SkipUI? 

If so, I think it's worth moving this to documentation and dropping the dependency. What do you think?

Come to think of it, we should probably add a general permission check utility to SkipUI itself... but that's another topic.

Reply:
Ah that dependency was vestigial! It's removed.

Reply:
Actually, it does look like we need to depend on SkipUI, since that brings in the compose dependencies we need for `androidx.activity.result.contract.ActivityResultContracts` and other things that are being used. That seems to be the only thing stopping CI from passing (https://github.com/skiptools/skip-bluetooth/actions/runs/10640472849), so if you add back the SkipUI dependency, then we should get a clean CI run.

Reply:
I'd rather not include the permission prompt util and not depend on SkipUI. What do you think?

Reply:
Oh yeah, if `askForBluetoothPermissions()` is removed, then it looks like the `androidx.activity.compose` and other imports can be removed, and it can just depend on skip-foundation.

Reply:
Removing that one function works for me. It's changed.

Reply:
I think you just need to remove the `import androidx.core…` and `import androidx.activity.compose…` from CBCentralManager.swift, and it looks like the CI run should pass (see the failure at https://github.com/skiptools/skip-bluetooth/actions/runs/10641143687).

Reply:
Ah I see. I wasn't checking the CI output -- fixed. Not sure what's causing the cla-bot failures though... Thank you for the help thus far!